driver/normalizer/parser.go

```diff
@@ -63,17 +63,18 @@ var ToNoder = &native.ObjectToNoder{
 		"Compare":  {"comparators": true, "ops": true},
 		// FIXME: check call.keywords
 		//"Call"                    : { "args": true, "keywords": true},
-		"With":                    {"body": true, "items": true},
-		"FunctionDef":             {"body": true, "decorator_list": true},
-		"Lambda":                  {"body": true},
-		"arguments":               {"defaults": true},
-		"Try":                     {"body": true, "orelse": true, "finalbody": true, "handlers": true},
```

Comment:
LGTM!

I am not very familiarized with the API to annotate nodes, out of curiosity, how is removing `handlers: true` affecting?